hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/safemode/ContainerSafeModeRule.java

```diff
@@ -63,19 +64,17 @@ public ContainerSafeModeRule(String ruleName, EventQueue eventQueue,
             " value should be >= 0.0 and <= 1.0");
 
     containerMap = new ConcurrentHashMap<>();
-    if(containers != null) {
-      containers.forEach(c -> {
-        // TODO: There can be containers in OPEN state which were never
-        // created by the client. We are not considering these containers for
-        // now. These containers can be handled by tracking pipelines.
-        if (c != null && c.getState() != null &&
-            !c.getState().equals(HddsProtos.LifeCycleState.OPEN)) {
-          containerMap.put(c.getContainerID(), c);
-        }
-      });
-      maxContainer = containerMap.size();
-    }
-
+    containers.forEach(container -> {
```

Comment:
Why containers!= null condition is removed?

Reply:
`ContainerManager.getContainers()` doesn't return null, it returns empty list if there are no containers.